src/orchard/meta.clj

```diff
@@ -146,6 +146,12 @@
   (concat (keys (var-get #'clojure.repl/special-doc-map))
           '[& catch finally]))
 
+(defn meta+
+  "Return special form or var's meta"
+  [v]
+  (or (special-sym-meta v)
```

Comment:
Hmm, I see we don't use this in `var-meta`. Perhaps its docstring is out of date and this used to be the same function.